loader/CMakeLists.txt

```diff
@@ -306,6 +306,9 @@ else()
     target_link_libraries(vulkan Vulkan::Headers)
     # Used to make alloca() and secure_getenv() available
     target_compile_definitions(vulkan PRIVATE _GNU_SOURCE)
+    if(CMAKE_SYSTEM_NAME MATCHES "FreeBSD")
+        add_compile_definitions(__BSD_VISIBLE=1)
```

Comment:
Was there a specific reason `add_compile_definitions` was used instead of `target_compile_definitions`? The latter prevents the macro from being applied to targets unnecessarily, and is preferred. (note that this projects CMake is objectively bad and I have 'rewrite cmake' on my todo list but haven't worked up it yet.)
```cmake
target_compile_definitions(vulkan PRIVATE __BSD_VISIBLE=1)
 ```

Reply:
Not really, and I've changed it, sorry! I agree that `target_compile_definitions` is better in this case.